solidity/compile.ts

```diff
@@ -19,6 +19,7 @@ const compilePetalLock = async () => {
 			'OpenRenewalManager.sol': { content: await fs.readFile('contracts/OpenRenewalManager.sol', 'utf8') },
 		},
 		settings: {
+			viaIR: true,
```

Comment:
What caused this change?

Reply:
Compiler error (/solidity/libsolidity/codegen/LValue.cpp:55):Stack too deep. Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON) while enabling the optimizer. Otherwise, try removing local variables.